bin/sl-runctl.js

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env node
-// Copyright IBM Corp. 2014,2015. All Rights Reserved.
+// Copyright Sam Roberts 2014,2016. All Rights Reserved.
```

Comment:
Ya, that was my mistake. It wasn't until I saw the result that I remembered the significance.